code/modules/mob/mob.dm

```diff
@@ -478,16 +474,6 @@
 /mob/proc/pull_damage()
 	return 0
 
-/mob/MouseDrop(mob/M as mob)
-	..()
-	if(M != usr) return
-	if(usr == src) return
-	if(!Adjacent(usr)) return
-	if(usr.incapacitated(INCAPACITATION_STUNNED | INCAPACITATION_FORCELYING | INCAPACITATION_KNOCKOUT | INCAPACITATION_RESTRAINED)) return //Incapacitated.

```

Comment:
It doesn't look like this particular check was transferred over to the new UI? Or is that handled by the tgui checks?

Reply:
If I'm not mistaken this is the code which opens the inventory when you click drag from a person (unless its you, out of range, incapped)? If so it seems to be handled elsewhere because the inventory does open correctly when you do that. I can double check the conditions tonight but it seems like if this check was entirely absent now then it wouldn't function at all.

Reply:
Ok yeah it's covered elsewhere. Now instead of not opening the inventory menu at all (except adjacency stuff), these checks are in TGUI and just don't let you interact with the menu (it's greyed out if you're, say, handcuffed)